.circleci/config.yml

```diff
@@ -0,0 +1,159 @@
+version: 2.1
+
+jobs:
+  ruby_style: &bazel_defaults
+    working_directory: /home/circleci/repo
+    resource_class: medium
+    docker:
+      - image: bazelruby/ruby-2.6.5
+    environment:
+      PATH: "/usr/local/bin:/usr/bin:/sbin:/opt/bin:/home/circleci/repo/bin:/bin:/sbin:/usr/sbin"
+      BUNDLE_PATH: /home/circleci/.bundle_cache
+      BAZEL_OPTS: "--host_jvm_args=-Xmx400m --host_jvm_args=-Xms400m"
+      BAZEL_BUILD_OPTS: "--curses=no --verbose_failures --jobs 10"
+      BAZEL_TEST_OPTS: "--verbose_failures --test_output=streamed --test_verbose_timeout_warnings "
+
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "Gemfile.lock" }}
+      - run:
+          name: Install Bundler
+          command: |
+            gem install bundler:2.0.2 --no-doc
+            bundle install --jobs=4 --retry=3 --path ${BUNDLE_PATH}
+
+      - save_cache:
+          paths:
+            - ${BUNDLE_PATH}
```

Comment:
For what purpose are you using bundler outside of Bazel's control?

Reply:
Only while we had issues building rubocop on Linux since it relies on another gem “jerry_wrinkle” with native extensions. Once this is fixed we won’t need to run rubocop without bazel.